packages/config/hackkit.config.ts

```diff
@@ -145,6 +145,11 @@ export default {
 		},
 	},
 	maxTeamSize: 4,
+	featureFlags: {
+		core: {
+			requireUsersApproval: false,
```

Comment:
Not a huge deal, but do you think this flag in particular with approving users might be useful to be something more dynamic like in the toggles vs in the config? I don’t have a strong opinion I’m just curious on your take